.github/workflows/update_libs_merge.yaml

```diff
@@ -0,0 +1,25 @@
+# reusable workflow for automatically approving PRs updating charm libraries
+name: Approve and Merge Updating Charm Libraries
+run-name: CI for approving and merging automated-update-charm-libs PRs
+
+on:
+  push:
+    branches:
+      - "automated-update-charm-libs"
```

Comment:
Have a look at the pr https://github.com/canonical/identity-platform-login-ui-operator/pull/59

adding the branch in there and then making sure that we have a branch protection rule set (check the settings for the renovate/* pattern) will do the job without an extra workflow